tools/qontract_cli.py

```diff
@@ -0,0 +1,72 @@
+import json
+import click
+
+import utils.gql as gql
+import utils.config as config
+import reconcile.queries as queries
+
+from reconcile.cli import config_file
+
+
+@click.group()
+@config_file
+@click.pass_context
+def root(ctx, configfile):
+    ctx.ensure_object(dict)
+    config.init_from_toml(configfile)
+    gql.init_from_config()
+
+
+@root.group()
+@click.pass_context
+def get(ctx):
+    pass
+
+
+@get.command()
+@click.pass_context
+def settings(ctx):
+    settings = queries.get_app_interface_settings()
+    print(json.dumps(settings))
```

Comment:
Can we use `logging` instead? For now, `print` is enough, but I see this growing and by then we will want `logging` being used everywhere.

Reply:
logging will not be picked up by the next tool, whatever that may be

Reply:
how so? logging can spit to stdout, just like print, as long as you set the streamhandler. Anyway, as we discussed, let's keep the effort to the minimum for now.